addons/website_slides/tests/test_slide_channel.py

```diff
@@ -274,6 +275,47 @@ def _get_completion_time_pdf(*args, **kwargs):
         self.assertEqual(13.37, round(slides_1.completion_time, 2))
         self.assertEqual(123.0, slides_2.completion_time)
 
+    def test_if_unlink_and_unpublish_channel_slides_sends_competition_email(self):
+        """When all content (slide.slide) are deleted from the course (slide.channel),
+        the course participants (slide.channel.partner) SHALL NOT receive completers email notification. """
+
+        with self.mock_mail_gateway():
+            _count_send_mails = lambda: len(self._new_mails.filtered(lambda m: m.model == 'slide.channel.partner'))
+
+            # 0. Setup: make sure there's channel with slides, that nobody completed
+            self.assertTrue(len(self.channel.slide_ids) > 1, "Channel shall have at least two slides.")
+            self.assertTrue(len(self.channel.channel_partner_ids) > 0, "Channel shall have at least one participant (channel_partner_ids)")
+            self.assertTrue(_count_send_mails() == 0, "Someone received competition mail notification (nobody should have)")
```

Comment:
In general, instead of asserting `True` for a comparison of integers, you should use 

```suggestion
            self.assertEqual(_count_send_mails(), 0, "Someone received competition mail notification (nobody should have)")
```

You'll get better feedback from the `unittest` module on failure: 
> 3 != 0

is better than 
> False is not True

But, as I said above, in this case where you only expect one record anyway, `assertFalse(_did_send_email_to_attendees(), ...)` will be more straightforward.

Reply:
Thanks for the feedback, I'll use boolean case